code/modules/medical/computer/cloning.dm

```diff
@@ -94,8 +94,9 @@
 				src.updateUsrDialog()
 				return 1
 
-/obj/machinery/computer/cloning/emag()
+/obj/machinery/computer/cloning/emag(mob/user)
 	emagged = 1
+	user.visible_message("<span class='warning'>[user.name] slides something into the [src.name]'s card-reader.</span>","<span class='warning'>You short out the [src.name].</span>")
```

Comment:
Instead of writing "The [src.name]", try "\The [src]" from now on. It actually works as you intend it and works with proper nouns